traffic_portal/test/end_to_end/users/users-spec.js

```diff
@@ -41,6 +57,84 @@ describe('Traffic Portal Users Test Suite', function() {
 		expect(first.isSelected()).toBe(false);
 		let tableColumns = element.all(by.css('#usersTable tr:first-child td'));
 		expect(tableColumns.count()).toBe(4);
+		first.click();
+	});
+
+	it('should open new users form page', function() {
+		console.log("Open new users form page");
+		browser.driver.findElement(by.name('createUserButton')).click();
+		expect(browser.getCurrentUrl().then(commonFunctions.urlPath)).toEqual(commonFunctions.urlPath(browser.baseUrl)+"#!/users/new");
+	});
+
+	it('should create a new user', function () {
+		console.log("Creating a new user");
+		expect(pageData.createButton.isEnabled()).toBe(false);
+		pageData.username.sendKeys(myNewUser.username);
+		pageData.fullName.sendKeys(myNewUser.fullName);
+		pageData.email.sendKeys(myNewUser.email);
+		pageData.roleName.sendKeys(myNewUser.roleName);
+		pageData.tenantId.sendKeys(myNewUser.tenantId);
```

Comment:
this is too fragile imo. i would do this instead

```
commonFunctions.selectDropdownbyNum(pageData.tenantId, 1);
```

to just select the first thing in the dropdown. also do this for role

Reply:
ok..